.benchmarks/README.md

```diff
@@ -0,0 +1,32 @@
+# Benchmarks
+
+This folder is where benchmarks are configured to be added on the dashboard generated in [benchmarks](https://gnoland.github.io/benchmarks).
```

Comment:
@moul @albttx
Can we host these at `benchmarks.gno.land`, or somewhere similar?

Reply:
good idea, let's do it